maintenance/errors/error.go

```diff
@@ -10,7 +10,7 @@ import (
 	"net/http"
 	"os"
 
-	raven "github.com/getsentry/raven-go"
+	"github.com/getsentry/raven-go"
```

Comment:
Even though the `raven` name is not necessary, if it differs from the name in the import path, you should still specify it. This is for readability.